lib/boom_notifier/webhook_notifier.ex

```diff
@@ -21,7 +22,7 @@ defmodule BoomNotifier.WebhookNotifier do
 
   import BoomNotifier.Helpers
 
-  @type options :: [{:url, String.t()}]
+  @type options :: [{:url, String.t()}, {:headers, Keyword.t()}]
```

Comment:
Since `headers` is an optional value, what do you think about doing something like this?

```elixir
@type options :: [{:url, String.t()}] | [{:url, String.t()}, {:headers, Keyword.t()}]
```

Reply:
Even better:

```elixir
  @type options ::
          [{:url, String.t()}] | [{:url, String.t()}, {:headers, HTTPoison.Request.headers()}]

```